src/lib/Libecl/ecl_verify.c

```diff
@@ -667,3 +702,63 @@ struct ecl_attribute_def *ecl_find_resc_def(
 	}
 	return NULL;
 }
+
+/**
+ * @brief
+ * 	Returns TRUE if the name passed in is an attribute.
+ *
+ * @note
+ * 	This must not be called with object of type MGR_OBJ_SITE_HOOK or MGR_OBJ_PBS_HOOK.
+ *
+ * @param[in]	object - type of object
+ * @param[in]	name  - name of the attribute
+ * @param[in]	attr_type  - type of the attribute
+ *
+ * @eturn int
+ * @retval	TRUE - means if the input is an attribute of the given 'object' type
+ *        	    and 'attr_type'.
+ * @retval	FALSE - otherwise.
+ *
+ */
+int
+is_attr(int object, char *name, int attr_type)
+{
+	struct ecl_attribute_def *attr_def = NULL;
+
+	if ((object == MGR_OBJ_SITE_HOOK) || (object == MGR_OBJ_PBS_HOOK)) {
+		return FALSE;
+	}
+
+	else if (object == MGR_OBJ_RSC) {
+		return TRUE;
+	}
```

Comment:
I'm not sure if it is a bug, it is sure weird.  If you think it is necessary, then it's OK.